app/services/theme_settings_migrations_runner.rb

```diff
@@ -57,7 +57,12 @@ def is_valid_url(url)
 
   def self.loader_js_lib_content
     @loader_js_lib_content ||=
-      File.read(File.join(Rails.root, "node_modules/loader.js/dist/loader/loader.js"))
+      File.read(
+        File.join(
+          Rails.root,
+          "app/assets/javascripts/discourse/node_modules/loader.js/dist/loader/loader.js",
```

Comment:
Oh, is it once again in non-root node_modules?

Reply:
Yeah, workspace package dependencies are no longer hoisted up to the root node_modules. So we have to find them in the package-specific node_modules directories